src/plugins/Polls/types.tsx

```diff
@@ -0,0 +1,10 @@
+export interface PollMetaData {
+    question: string
+    key: string | number | symbol
```

Comment:
Metadata will be encoded into the post therefore it isn't possible to store a symbol in it.